deployments/liqo/templates/liqo-auth-deployment.yaml

```
@@ -70,25 +70,31 @@ spec:
           command: ["/usr/bin/auth-service"]
           args:
           - --namespace=$(POD_NAMESPACE)
-          - --resyncSeconds=30
-          - --enableAuthentication={{ .Values.auth.config.enableAuthentication }}
+          {{- if not .Values.auth.tls}}
+          - --address=:5000
+          {{- else }}
+          - --address=:8443
+          - --enable-tls
+          {{- end }}
+          - --enable-authentication={{ .Values.auth.config.enableAuthentication }}
+          {{- if .Values.apiServer.address }}
+          - --advertise-api-server-address={{ .Values.apiServer.address }}
+          - --advertise-api-server-trusted-ca={{ .Values.apiServer.trustedCA }}
+          {{- end }}
```

```suggestion
          {{- if .Values.apiServer.address }}
          - --advertise-api-server-address={{ .Values.apiServer.address }}
          {{- end }}
          {{- if .Values.apiServer.trustedCA }}
          - --advertise-api-server-trusted-ca={{ .Values.apiServer.trustedCA }}
          {{- end }}
```

It's probably clearer and more flexible